src/vorta/views/main_window.py

```diff
@@ -156,19 +167,17 @@ def profile_add_action(self):
             self.profileSelector.setCurrentIndex(self.profileSelector.currentIndex())
 
     def backup_started_event(self):
-        self.set_status(progress_max=0)
         self._toggle_buttons(create_enabled=False)
+        self.set_log('')
 
```

Comment:
```suggestion
        font = self.progressText.font()
        font.setBold(True)
        self.progressText.setFont(font)

```

Reply:
The font sizes seem funny on Linux. "Start Backup" button seems too small.

Reply:
I'm also not happy with the button changing size. Better we use a min-size again.

Reply:
Made it non-bold in standard font size. Logs below slightly smaller. Hope this doesn't cause issues on Linux. Min size for the button.

![Screen Shot 2020-09-05 at 19 28 11](https://user-images.githubusercontent.com/3916435/92304147-024a6780-efae-11ea-94d7-a3edae381373.jpg)